gtsfm/common/two_view_estimation_report.py

```diff
@@ -4,12 +4,30 @@
 """
 
 from dataclasses import dataclass
+from enum import Enum
 from typing import Optional
 
 import numpy as np
 from gtsam import Rot3, Unit3
 
 
+class TwoViewConfigurationType(str, Enum):
+    """Categorization of the configuration of two views.
+
+    Calibrated indicates that the two-view relation has sufficient parallax, and intrinsics are known and reliable.
+    Uncalibrated indicates that the two-view relation has sufficient parallax, but intrinsics are unknown / unreliable.
+    Planar / Panoramic indicates that the relation is a homography.
+    Degenerate indicates that there was insufficient support to accurately determine
+        the two-view relation.
+    """
```

Comment:
I'd prefer the documentation for each enum entry to be placed above it, instead of a paragraph.